A way to add arguments to MD_RENDERER

I'd like to customize the call to `pandoc`, e.g. by using a template:

```
pandoc --template my_template.md
```

I tried doing:

```
export MD_RENDERER="pandoc --template my_template.md"
```

but it fails for [this reason](https://unix.stackexchange.com/q/444946/417321).

Therefore I think an additional option called `MD_RENDERER_ARGS` could be added (just my first idea, maybe there's a better way).

Sorry for responding after some time, I was rather busy.

I see the problem, I believe that it can be fix with using `eval` – something like:

```diff
diff --git a/ssb b/ssb
index e5186b9..4a6cdc5 100755
--- a/ssb
+++ b/ssb
@@ -304,7 +304,7 @@ make_html_files()
 		# Ignore the suggestion because tempf can be expanded immediately
 		# shellcheck disable=SC2064
 		trap "rm $tempf" 0 2 3 15
-		append_posts_list "$posts" | cat "$md_file" - | "$MD_RENDERER" > "$tempf"
+		append_posts_list "$posts" | cat "$md_file" - | eval "$MD_RENDERER" > "$tempf"
 		cat "$HEADER_PATH" "$tempf" "$FOOTER_PATH" > "$output_file"
 	done
}
```

I tested it in a very ad hoc way – seems to work. If you have time you can check out if it works in your scenario, then please let me know.

If it works and you are willing, you can open a PR to the development branch and to the honours.

--- a/ssb
+++ b/ssb
@@ -304,7 +304,7 @@ make_html_files()
 		# Ignore the suggestion because tempf can be expanded immediately
 		# shellcheck disable=SC2064
 		trap "rm $tempf" 0 2 3 15
-		append_posts_list "$posts" | cat "$md_file" - | "$MD_RENDERER" > "$tempf"
+		append_posts_list "$posts" | cat "$md_file" - | eval "$MD_RENDERER" > "$tempf"
 		cat "$HEADER_PATH" "$tempf" "$FOOTER_PATH" > "$output_file"
 	done
 }

